modules/samples/producers/wso2-event-performance/src/main/java/org/wso2/carbon/sample/performance/Client.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.sample.performance;
+
+import org.apache.log4j.Logger;
+import org.wso2.carbon.databridge.agent.AgentHolder;
+import org.wso2.carbon.databridge.agent.DataPublisher;
+import org.wso2.carbon.databridge.commons.Event;
+
+import java.text.DecimalFormat;
+import java.util.Arrays;
+import java.util.Random;
+
+public class Client {
+    private static Logger log = Logger.getLogger(Client.class);
+
```

Comment:
Hi Tharik,

Please find my responses below.

1. Here, warm up is actually for the server. AFAIK, usually it is recommended to warm up for at least 10-15 min before running the actual perf test. Following links [1,2] describe about warm up time for java perf tests.
2. Well I agree that nano second accuracy might not be needed for the current tests we run. But it might be useful if we use this client is used for more complex calculations. This is just a suggestion in case it is needed for future tests. But if there is a significant cost associated with adding nano time to the stream, I guess we can keep it in millis as it is.

[1] http://stuq.nl/weblog/2009-01-28/why-many-java-performance-tests-are-wrong
[2] https://www.xing.com/communities/posts/important-aspects-in-load-testing-server-warmup-1003946221